test/unit_linalg_solver.ml

```diff
@@ -0,0 +1,92 @@
+open Owl
+open Owl_exception
+
+module N = Owl_base_dense_ndarray.D
+module L = Owl_base_linalg_generic
+
+(* define the test error *)
+
+let approx_equal a b =
```

Comment:
Maybe we should put this in `Maths` instead of redefining it over and over in different modules

Reply:
Thanks! I replace the implementation with `approx_equal` function in the dense module.